.github/workflows/release-orchestrate-overall.yml

```diff
@@ -12,7 +12,22 @@ on:
         required: true
         default: '1,2,3,4,5,6,7'
 jobs:
+  pre_job:
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
```

Comment:
should use a SHA instead of branch to avoid potential for malicious code

Reply:
added